mindmeld/cli.py

```diff
@@ -811,6 +813,112 @@ def augment(app_path, language):
     logger.info("Augmentation Complete.")
 
 
+@shared_cli.command("active_learning", context_settings=CONTEXT_SETTINGS)
+# Params Used for Both Select and Train
+@click.option("--app-path", type=str, help="Path to the MindMeld application")
+@click.option(
+    "--batch_size", type=int, help="Number of queries to select each iteration."
+)
+@click.option(
+    "--training_level",
+    type=str,
+    help="The hierarchy level to train ('domain' or 'intent').",
+)
+@click.option(
+    "--output_folder",
+    type=str,
+    help="Folder to store output.",
+)
+# Params Specific to Training
+@click.option(
+    "--train",
```

Comment:
Didn't we decide to change this to "tune" or something?

Reply:
Will make that change shortly

Reply:
Changed